### ISSUE TITLE
Add nolinebreak setting

### ISSUE THREAD

**Issue by antofthy** [qnighy/vim-ssh-annex#2]
I also suggest adding... to the syntax files...
```
" No word wrapping on lines in this file (does not make sense)
setlocal nolinebreak
```
I have linebreak (line break on whitespace or word wrapping) to allow easier reading of comments and plain text files for most things.  But that really does not make any sense with these files (unless you can specify line break points as part of syntax highlighting, which I don't think you can.

Gosh... Syntactic line breaks for something like JSON files (which are otherwise all on one line) would be a great feature!

**Comment by qnighy** [comment]
Such configurations are up to user, so I don't think it should be included in this plugin. If there's an interface for providing break hints, that would better fit in this plugin though.

**Comment by antofthy** [comment]
You are right..  I moved that configuration to the "ftplugin/" directory in a file of the same name.
Though rather than set nolinebreak, I now enable "paste mode" for that file, made more sense for me.
